### ISSUE TITLE
Add unit tests for download_file command

### ISSUE THREAD

**Issue by RishabhJain2018** [Cloud-CV/evalai-cli#157]

**Comment by kurianbenoy** [comment]
Can I work on this issue?

**Comment by pushkalkatara** [comment]
Hi @RishabhJain2018 Can i work on this issue?

**Comment by RishabhJain2018** [comment]
Sure @pushkalkatara

**Comment by pushkalkatara** [comment]
@RishabhJain2018 I am unable to login into the CLI using the ID: host and Password; password. It says invalid credentials. What might be the issue?

**Comment by pushkalkatara** [comment]
I have written the tests but unable to run them due to no login. Here's the commit - https://github.com/pushkalkatara/evalai-cli/commit/f4b511750ff24a8c673b0460cb0f470b2a2f41d5

**Comment by RishabhJain2018** [comment]
> @RishabhJain2018 I am unable to login into the CLI using the ID: host and Password; password. It says invalid credentials. What might be the issue?

Please check that you're running EvalAI locally as well and host url is set to `localhost`.

**Comment by pushkalkatara** [comment]
yes host url is set to localhost:8888 and the EvalAI is running locally. Still when i login using credentials ID: host and password: password, it just exits without any message. I think it should give invalid credentials or login successful.
```
(venv) (base)  ~/cv/dev/evalai-cli: evalai login 
username: host
Enter password: 
```

**Comment by RishabhJain2018** [comment]
The host URL should be set to `localhost:8000` as CLI interacts with the API.

**Comment by pushkalkatara** [comment]
The pytests worked, but still, login as a host or a participant is not working.

**Comment by pushkalkatara** [comment]
@RishabhJain2018 I solved the issue and created a pull request. Please review the PR #173 .
